README.adoc

```diff
@@ -2,188 +2,184 @@
-activates the Netflix Eureka `DiscoveryClient` implementation. (There are other
-implementations for other service registries, such as
-https://www.consul.io/[Hashicorp's Consul] or https://zookeeper.apache.org/[Apache Zookeeper]).
-The following listing (from
-`eureka-client/src/main/java/example/serviceregistrationanddiscoveryclient/ServiceRegistrationAndDiscoveryClientApplication.java`)
-shows the client application:
+Now that you have started a service registry, you can stand up clients that interact with the registry. Our client applications, ServiceA and ServiceB, will automatically register with the Eureka server because we have `spring-cloud-starter-netflix-eureka-client` on the classpath.
+To avoid port conflicts, set the `server.port` parameter in both ServiceA and ServiceB:
 
+Service A:
 ====
-[source,java]
+[source,properties]
 ----
-include::complete/eureka-client/src/main/java/com/example/serviceregistrationanddiscoveryclient/ServiceRegistrationAndDiscoveryClientApplication.java[]
+include::servicea/src/main/resources/application.properties[]
 ----
 ====
 
-Whatever implementation you choose, you should soon see `eureka-client` registered under
-whatever name you specify in the `spring.application.name` property. This property is used
-a lot in Spring Cloud, often in the earliest phases of a service's configuration. This
-property is used in service bootstrap and, so, by convention lives in
-`eureka-client/src/main/resources/bootstrap.properties` where it is found before
-`src/main/resources/application.properties`. The following listing shows the
-`bootstrap.properties` file:
-
+Service B:
 ====
 [source,properties]
 ----
-include::complete/eureka-client/src/main/resources/bootstrap.properties[]
+include::serviceb/src/main/resources/application.properties[]
 ----
 ====
 
-The `eureka-client` defines a Spring MVC REST endpoint (`ServiceInstanceRestController`)
-that returns an enumeration of all the registered `ServiceInstance` instances at
-`http://localhost:8080/service-instances/a-bootiful-client`. See the
-https://spring.io/guides/gs/rest-service/[Building a RESTful Web Service]  guide to learn
-more about building REST services with Spring MVC and Spring Boot.
+At this point you should be able to run all three applications.
+You can use the IDE or execute the command `./gradlew bootRun` from each application folder.
 
-== Test the Application
+When the applications are running, you can view the http://localhost:8761/[Eureka dashboard^].
+
+image::eureka-server-dashboard.png[]
 
-Test the end-to-end result by starting the `eureka-service` first and then, once that has
-loaded, starting the `eureka-client`.
+== Service A Endpoint
 
-To run the Eureka service with Maven, run the following command in a terminal window (in
-the `/complete` directory):
+Create a new class `com/example/servicea/controller/ServiceARestController.java` in the `servicea` project that exposes an endpoint you will use to test the application.
```

Comment:
Add "called" after "class"

Change "that exposes" to "to expose"

Change "will" to "can"